training-net/src/Startup.cs

```diff
@@ -36,7 +37,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
             services.AddDbContext<MvcMovie.Repositories.Database.DataBaseContext>(options =>  options.UseNpgsql(Configuration["MvcMovieContext"]));
             services.AddScoped<MvcMovie.Repositories.Database.DataBaseContext>();
-
+            services.AddJsonLocalization(options => options.ResourcesPath = "Resources");
+            services.AddMvc().AddViewLocalization();
+            CultureInfo.CurrentCulture = new CultureInfo("en-US");

```

Comment:
here you have to set `CultureInfo.CurrentUICulture` instead of `CultureInfo.CurrentCulture`